client/src/pages/Login.js

```diff
@@ -0,0 +1,193 @@
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import InputBase from '@material-ui/core/InputBase';
+import Divider from '@material-ui/core/Divider';
+
+import GoogleLogin from 'react-google-login';
+import Cookies from 'universal-cookie';
+
+import Logo from '../images/logo.png';
+
+const BootstrapInput = withStyles((theme) => ({
+    root: {
+        'label + &': {
+            marginTop: theme.spacing(3),
+        },
+    },
+    input: {
+        borderRadius: 4,
+        position: 'relative',
+        backgroundColor: theme.palette.common.white,
+        border: '1px solid #ced4da',
+        fontSize: 16,
+        width: '100%',
+        height: 22,
+        padding: '10px 12px',
+        textAlign: 'center',
+        transition: theme.transitions.create(['border-color', 'box-shadow']),
+    },
+}))(InputBase);
+
+const useStyles = makeStyles((theme) => ({
+
+    logo: {
+        height: '24px',
+        margin: '25px auto'
+    },
+
+    shadowCard: {
+        width: 390,
+        border: 'none',
+        boxShadow: '0px 0px 15px 0px rgba(215,214,235,1)',
+        textAlign: 'center'
+    },
+
+    textOrange: {
+        color: 'rgb(255, 125, 19)',
+        textDecorationLine: 'none'
+    },
+
+    title: {
+        margin: '40px auto',
+        transform: 'scale(1.12,1)'
+    },
+
+    label: {
+        fontSize: '12px',
+        fontWeight: 600
+    },
+
+    input: {
+        margin: '10px 0',
+    },
+
+    body: {
+        padding: '5px 50px'
+    },
+
+    btnOrange: {
+        backgroundColor: 'rgb(255, 125, 19)',
+        color: '#fff',
+        fontWeight: 300,
+        textTransform: 'none',
+        margin: '40px auto',
+        padding: '8px 30px',
+        '&:hover': {
+            backgroundColor: 'darkorange'
+        }
+    },
+
+    textFooter: {
+        fontWeight: 500
+    },
+
+}));
+
+function Login() {
+    const classes = useStyles();
+    const [confirmed, setConfirmed] = React.useState(false);
+    const [email, setEmail] = React.useState('');
+
+    const handleChange = (e) => {
+        setEmail(e.target.value)
+    }
+
+    const handleGetStarted = () => {
+        setConfirmed(true)
+    }
+
+    const responseGoogle = (response) => {
+        let status;
+        fetch("/googlelogin", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify({ tokenId: response.tokenId })
+        })
+            .then(res => {
+                status = res.status;
+                if (status < 500) return res.json();
+                else throw Error("Server error");
+            })
+            .then(res => {
+                if (status === 200) alert(res.response);
+                const cookies = new Cookies();
+                cookies.set('token', res.token, { path: '/', httpOnly: true })
+                if (status === 401) alert(res.response);
```

Comment:
Hi! I'm assuming you'll change these `alert`(s) in the future to do something else with the response?

Reply:
Exactly!